### PR TITLE
Release notes for 3.12.14 and 4.1.4

### PR DESCRIPTION
**Pull request by thotz**
Signed-off-by: Jiffin Tony Thottan <jthottan@redhat.com>

### CODE REVIEW

File: docs/release-notes/3.12.14.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Major changes, features and limitations addressed in this release
+This release contains a fix for a security vulerability in Gluster as follows,
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10907
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10904
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10911
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10913
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10923
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10930
+
+Plus to resolve one of the security vulerability following limitations were made
```
Comment (humblec) [inline docs/release-notes/3.12.14.md]: typo on vulerability

File: docs/release-notes/4.1.4.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Release notes for Gluster 4.1.4
+
+This is a bugfix release. The release notes for [4.1.0](4.1.0.md),
+ [4.1.1](4.1.1.md), [4.1.2](4.1.2.md) and [4.1.3](4.1.3.md) contains a
+listing of all the new features that were added and bugs fixed in the
+GlusterFS 4.1 stable release.
+
+## Major changes, features and limitations addressed in this release
+
+This release contains a fix for a security vulerability in Gluster as follows,
```
Comment (humblec) [inline docs/release-notes/4.1.4.md]: same typo.

File: docs/release-notes/4.1.4.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10907
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10904
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10911
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10913
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10923
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10930
+
+Plus to resolve one of the security vulerability following limitations were made
```
Comment (humblec) [inline docs/release-notes/4.1.4.md]: same here

**Comment by humblec** [comment]
@ShyamsundarR please review :)

**Comment by thotz** [comment]
Typos addressed

File: docs/release-notes/4.1.4.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Release notes for Gluster 4.1.4
+
+This is a bugfix release. The release notes for [4.1.0](4.1.0.md),
+ [4.1.1](4.1.1.md), [4.1.2](4.1.2.md) and [4.1.3](4.1.3.md) contains a
+listing of all the new features that were added and bugs fixed in the
+GlusterFS 4.1 stable release.
+
+## Major changes, features and limitations addressed in this release
+
+This release contains a fix for a security vulnerability in Gluster as follows,
```
Comment (prashanthpai) [inline docs/release-notes/4.1.4.md]: for following a security vulnerabilities

same comment for the other file

File: docs/release-notes/4.1.4.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10913
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10923
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10930
+
+Plus to resolve one of the security vulnerability following limitations were made
+- open,read,write on special files like char and block are no longer permitted
+- io-stat xlator can dump stat info only to /var/run/gluster directory
+
+Installing the updated packages and restarting gluster services on gluster
+brick hosts, will help prevent the security issue.
```
Comment (prashanthpai) [inline docs/release-notes/4.1.4.md]: fix the above security issues 

same comment for the other file

File: docs/release-notes/4.1.4.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+This release contains a fix for a security vulnerability in Gluster as follows,
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10907
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10904
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10911
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10913
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10923
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10930
+
+Plus to resolve one of the security vulnerability following limitations were made
```
Comment (prashanthpai) [inline docs/release-notes/4.1.4.md]: To resolve, one of the security vulnerabilities

same comment for the other file

File: docs/release-notes/3.12.14.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10904
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10911
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10913
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10923
+- https://nvd.nist.gov/vuln/detail/CVE-2018-10930
+
+Plus to resolve one of the security vulnerability following limitations were made
+- open,read,write on special files like char and block are no longer permitted
+- io-stat xlator can dump stat info onlyt to /var/run/gluster directory
+- Resolved with SSL/TLS encryption with kvm
```
Comment (prashanthpai) [inline docs/release-notes/3.12.14.md]: What does this line 21 mean?

**Comment by thotz** [comment]
Address the comments

**Pull request merged**